src/parsing/ris_live.go

```diff
@@ -0,0 +1,46 @@
+package parsing
+
+import (
+	"BGPAlert/common"
+	"encoding/json"
+	"fmt"
+	"log"
+
+	"github.com/gorilla/websocket"
+)
+
+func main() {
+	// RIS Live websocket url
+	socketUrl := "ws://ris-live.ripe.net/v1/ws/"
+
+	// create websocket connection to ris live websocket
+	conn, _, err := websocket.DefaultDialer.Dial(socketUrl, nil)
+	if err != nil {
+		log.Fatal("Websocket connection error:", err)
+	}
+	defer conn.Close()
```

Comment:
I think this line should go directly under where you `Dial`.  So when something weird happens in the middle, your program always successfully closes the connection.